core/lib/types/src/storage_writes_deduplicator.rs

```diff
@@ -3,14 +3,18 @@ use std::collections::HashMap;
 use zksync_utils::u256_to_h256;
 
 use crate::tx::tx_execution_info::DeduplicatedWritesMetrics;
+use crate::writes::compression::compress_with_best_strategy;
+use crate::writes::{BYTES_PER_DERIVED_KEY, BYTES_PER_ENUMERATION_INDEX};
 use crate::{AccountTreeId, StorageKey, StorageLogQuery, StorageLogQueryType, U256};
 
-#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq, Default)]
 pub struct ModifiedSlot {
     /// Value of the slot after modification.
     pub value: U256,
     /// Index (in L1 batch) of the transaction that lastly modified the slot.
     pub tx_index: u16,
+    /// Size of update in bytes
```

Comment:
I would also emphasize that it is the size that it will take up in pubdata (it might not be obvious for a person without context)